core/src/com/unciv/ui/cityscreen/CityScreen.kt

```diff
@@ -196,6 +201,11 @@ class CityScreen(
         }
     }
 
+    private fun playEnterCitySound() {
+        SoundPlayer.clearCache()
```

Comment:
how can I only stop this sound instead of stopping them all? Also I doubt clearing the cache is even a good way of stopping the sounds as they would need to be cached again, right?

Reply:
better to move this to the mod

Reply:
https://freesound.org/people/addiofbaddi/sounds/241683/

Reply:
clearCache is public for game shutdown, _not_ normal operations. At the moment you can't stop a sound, just as looping Sounds are no-go: They're _supposed_ to be tiny and short (anything > 1s would be better off streaming, there's Gdx.Music for that). Which is why SoundPlayer doesn't bother exposing the Gdx Sound.loop/pause/resume/stop methods.

If you need stop functionality, extend the helper class - but then you would need to learn why Gdx has two entirely different concepts of what a sound "instance" means - is the Java instance relevant or the long ID? Or are they interchangeable? Inconsistent.